sky/provision/__init__.py

```diff
@@ -36,13 +36,27 @@ def _wrapper(*args, **kwargs):
 # TODO(suquark): Bring all other functions here from the
 
 
+def stop_or_terminate_instances(provider_name: str,
```

Comment:
why combine them? I think we should call either stop or terminate explicitly, even if they are often been used together. this is an API instead of simply a util library

Reply:
I found that we have to do the same `if ... else ...` conditions for the stop/termination cases in the two places. I am ok to have them in the caller. Just changed it back. Thanks!